researcher_ids/UPDATE_PROCEDURE.md

```diff
@@ -0,0 +1,33 @@
+# Update Procedures
+
+This document outlines the maintainers and update procedures to suggest and innitiate changes to the GA4GH Researcher Identities & Access Claims [specifications](http://bit.ly/ga4gh-ri-v1) and documentation.
+
+## Specification maintainers
+
+**Name**: Craig Voisin  
+    Host Institute: Google  
+    Email Address: craigv@google.com  
+    GitHub: @cdvoisin  
+
+**Name**: Mikael Linden  
+    Host Institute: CSC  
+    Email Address: mikael.linden@csc.fi  
+    GitHub:  
+
+## Specifications updates
+
+The GA4GH Researcher Identities & Access Claims standard is established by members of the Data Use and Research Indentities (DURI) work stream. 
+
+To suggest or innitiate changes to the [specifications](http://bit.ly/ga4gh-ri-v1) please first contact the specification maintainers.
+
+Prior to recommending changes please:
+  * refer to the [landing page](https://github.com/ga4gh-duri/ga4gh-duri.github.io/tree/master/researcher_ids) for additional information about the purpose and use of the standard.
+  * identify and provide the following information:
+      1. Your name and affiliation
+      2. If you are currently a member of GA4GH and in what capacity
+      2. If you are currently using an implementation of RI Claims
+      3. If you are recommending a change to an implementation, the specifications, or the documentation
```

Comment:
Perhaps this is too much of a sentence fragment, so I'm having trouble parsing the request. I think this means to say:

Indicate whether you are recommending the change on behalf of one or more of the following: an implementation of the spec, the RI Claims specification itself, or other documentation in the researcher_ids directory.